src/dclass.d

```diff
@@ -1415,18 +1426,52 @@ public:
      */
     final bool isAbstract()
     {
-        if (isabstract)
-            return true;
+        if (isabstract != ABSfwdref)
+            return isabstract == ABSyes;
+
+        /* Bugzilla 11169: Resolve forward references to all class member functions,
+         * and determine whether this class is abstract.
+         */
+        extern (C++) static int func(Dsymbol s, void* param)
+        {
+            auto fd = s.isFuncDeclaration();
+            if (!fd)
+                return 0;
+            if (fd.storage_class & STCstatic)
+                return 0;
+
+            if (fd._scope)
+                fd.semantic(null);
```

Comment:
This causes a forward reference bug, https://issues.dlang.org/show_bug.cgi?id=16273